geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -380,19 +371,39 @@ function Home({
                     style={theme?.languageSelector?.style}
                 />}
             />
-            <ConnectedCardGrid
-                user={user}
-                query={query}
-                pageSize={pageSize}
-                isColumnActive={!!resource}
-                containerStyle={!isHeroVisible
-                    ? {
-                        marginTop: dimensions.brandNavbarHeight,
-                        minHeight: `calc(100vh - ${dimensions.brandNavbarHeight + dimensions.menuIndexNodeHeight + dimensions.footerNodeHeight}px )`,
-                        paddingBottom: dimensions.footerNodeHeight
-                    }
-                    : undefined}
-                column={
+            <div className="gn-main-home container-fluid">
+                <div className="row">
+                    <div className={`col-md-3 col-sm-12 m-3 ${ !showFilterForm ? 'collapse' : ''}`}>
```

Comment:
I think we could use directly the react-bootstrap components if they fit our needs.
https://react-bootstrap.github.io/layout/grid/
https://react-bootstrap.github.io/components/alerts/

Note that with this layout there are a misalignement between contents and headers, in addition the datail panel has some margin on the left

- header content misalignement
![image](https://user-images.githubusercontent.com/19175505/112812943-f6bfaa80-907d-11eb-8562-1ffc39fd60f3.png)

- padding on detail panel
![image](https://user-images.githubusercontent.com/19175505/112812771-c7a93900-907d-11eb-9dcd-769d43d4f8e1.png)